### ISSUE TITLE
use isort pre-commit (https://github.com/timothycrosley/isort) as a sorting tool of imports

### ISSUE THREAD

**Issue by zhukovgreen** [oamg/leapp-repository#564]
# Blocked by #562 (line length)

Under the scope:
- [ ] setup isort pre-commit hook  (https://github.com/timothycrosley/isort) with the following configuration

in pyproject.toml
```toml
[tool.isort]
line_length = {project-accepted-line-length}
use_parentheses = true
balanced_wrapping = true
include_trailing_comma = true
multi_line_output = 3
known_third_party=["pytest", "distro", "pyudev", "mock"]
```
- [ ] apply the hook to the whole codebase

**Comment by pirat89** [comment]
we are alredy checking isort for the updated files and providing `make lint_fix` rule that handles it automatially when user call it. However, we do not want to apply isort globally always, as sometimes is making the code less readable (e.g. switching the block/multiiline import by singleline import, ..). Also this is very low priority and as it has not been touched in 2.5y and it doesn't seem we will update it in a close future, I am closing it.
